agent/src/agent/pipeline/config/jython_scripts/test_pipelines/influx.py

```diff
@@ -39,22 +40,35 @@ def influx_request(base_url, url, method='GET', params=None, data=None, ssl=Fals
 try:
     influx_url_parsed = urlparse(sdc.userParams['HOST'])
     ssl = True if influx_url_parsed.scheme == 'https' else False
-    # get list of databases
-    params = {'q': 'SHOW DATABASES', 'db': None}
+    if sdc.userParams['MEASUREMENT_NAME']:
+        params = {'q': 'SELECT * FROM ' + sdc.userParams['MEASUREMENT_NAME'], 'db': sdc.userParams['DATABASE']}
+    elif sdc.userParams['QUERY']:
+        params = {'q': sdc.userParams['QUERY'].replace('{TIMESTAMP_CONDITION}', '1=1'), 'db': sdc.userParams['DATABASE']}
+    else:
```

Comment:
we should always have either QUERY or MEASUREMENT_NAME. The only case when it's not defined is when you create or edit a source. In that case, when creating a test pipeline we can set the `SHOW DATABASES` and db `null` there and get rid of these else conditions